src/tools/csi-kata-directvolume/examples/pod-with-directvol3/csi-app.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: spdk-pod-test
+spec:
+  runtimeClassName: kata  
+  containers:
+  - name: app
+    image: docker.io/library/ubuntu:22.04
+    command: ["sleep", "3600"]
+    volumeMounts:
+    - name: data
+      mountPath: /data
+  volumes:
+  - name: data
+    persistentVolumeClaim:
+      claimName: why1-spdk-dd-pvc
```

Comment:
What does this name mean? Please use a name associated with Kata, for example, `kata-sdpk-directvolume-pvc`.